### PR TITLE
Replace uses of the word "sane"

### PR DESCRIPTION
**Pull request by angrycub**
Switching to more inclusive language by replacing "sane" with "reasonable".

### CODE REVIEW

File: manager/runner.go [hunk 1]
```diff
@@ -27,9 +27,9 @@ import (
 )
 
 const (
-	// saneViewLimit is the number of views that we consider "sane" before we
```
Comment (eikenb) [inline manager/runner.go:30]: IMO both of these variable names are to long. What do you think of simply `viewLimit`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -482,7 +482,7 @@ BUG FIXES:
 
 BREAKING CHANGES:
 
-  * Retry now has a sane maximum default. Previous versions of Consul Template
+  * Retry now has a reasonable maximum default. Previous versions of Consul Template
```
Comment (eikenb) [inline CHANGELOG.md:485]: In line with my comment below... this use of 'reasonable' becomes optional but would still make sense.

Comment (angrycub) [inline CHANGELOG.md:485]:
```suggestion
  * Retry now has a maximum default. Previous versions of Consul Template
```

File: manager/runner.go [hunk 3]
```diff
@@ -27,9 +27,9 @@ import (
...
+	// reasonableViewLimit is the number of views that we consider reasonable before we
```
Comment (angrycub) [inline manager/runner.go:30]:
```suggestion
	// viewLimit is the number of views that we consider reasonable before we
```

File: manager/runner.go [hunk 4]
```diff
@@ -27,9 +27,9 @@ import (
...
 	// warn the user that they might be DDoSing their Consul cluster.
-	saneViewLimit = 128
+	reasonableViewLimit = 128
```
Comment (angrycub) [inline manager/runner.go:32]:
```suggestion
	viewLimit = 128
```

File: manager/runner.go [hunk 5]
```diff
@@ -276,7 +276,7 @@ func (r *Runner) Start() {
 
 	for {
 		// Warn the user if they are watching too many dependencies.
-		if r.watcher.Size() > saneViewLimit {
+		if r.watcher.Size() > reasonableViewLimit {
```
Comment (angrycub) [inline manager/runner.go:279]:
```suggestion
		if r.watcher.Size() > viewLimit {
```

**Review by eikenb — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**